storage/api/Storage.Samples/SetAutoclass.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_set_autoclass]
+
+using Google.Cloud.Storage.V1;
+using Google.Apis.Storage.v1.Data;
+using static Google.Apis.Storage.v1.Data.Bucket;
+using System;
+
+public class SetAutoclassSample
+{
+    public Bucket SetAutoclass(string bucketName, bool enable)
+    {
+        // Note: Only patch requests that disable autoclass are currently supported.

```

Comment:
Why receive the parameter then, that's very unclear for users. It'll be clearer if we do directly `Enabled = false`.

If patch ever supports enabling autoclas, we'll have to change this sample to remove this note, and we could add a parameter at that point. I'd prefer if we don't have a parameter now.

@cojenco any thoughts?

Reply:
Makes sense, let's remove the parameter for now. 

We were planning ahead for v2 that supports enablement on existing buckets. You're right that we'll have to update this sample anyhow at that point. I'll add a note in the tracking doc, thanks!